Scripts/UpdateMFSGlobals.py

```diff
@@ -0,0 +1,210 @@
+# Update the globals that are updated by the Master File Server
+# in a running instance of VistA.
+#
+# The list of globals to update was taken from
+# http://www.vistapedia.com/index.php/MFS_list
+#
+# For detailed help information, please run
+#   python UpdateMFSGlobals.py -h
+#
+#---------------------------------------------------------------------------
+# Copyright 2016 The Open Source Electronic Health Record Agent
```

Comment:
Copyright needs to be updated.

Reply:
Thanks Sam. I'll fix this in a new commit